naga/src/proc/mod.rs

```diff
@@ -2,6 +2,7 @@
 [`Module`](super::Module) processing functionality.
 */
 
+mod builtins;
```

Comment:
Should this be named `overloads`? It's more about overload sets than builtin functions, specifically.